geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
+Ag...N      2.15444(6)
+N    H      1.0129
+H    N...Ag 113.48(2)
+
+# Bond lengths: rs     (angstroms)
+
+Ag   Cl     2.2633(9)
+Ag...N      2.1545(8)
+
+NAME = AgH3IN
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Medcraft C, Gougoula E, Bittner DM, Mullaney JC, Blanco S, Tew DP, Walker NR, Legon AC (2017)
+Molecular geometries and other properties of H2O⋅⋅⋅AgI and H3N⋅⋅⋅AgI as characterized by rotational
+spectroscopy and ab initio calculations. J Chem Phys 147(23):234308/1-234308/8
+https://doi.org/10.1063/1.5008744
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   I   2.5375(3)
+Ag...N   2.180(1)
+Ag   N H 110.86(5)
+
+# Bond lengths: rs     (angstroms)
+# Bond angles:  thetas (degrees)
+
+Ag...N   2.182(1)
+Ag   N H 110.93(3)
+
+NAME = AgH3IP
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Stephens SL, Tew DP, Walker NR, Legon AC (2016) H3P⋅⋅⋅AgI: generation by laser-ablation and
+characterization by rotational spectroscopy and ab initio calculations. Phys Chem Chem Phys 18(28):18971-
+18977
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag  I   2.5483(1)
+P...Ag  2.3488(20)
+P   H   1.4086
+H   P H 118.92 
+
+NAME = AlBr3
+# Point Group: D3h
+# Shape: Trigonal planar
+# Bond lengths: rg(a)     (angstroms)
+# Source Paper: a. Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
+electron diffraction and computation. Struct Chem 23 (3):879-893
+b. Hargittai M, Kolonits M, Gödörházy L (1996) Molecular geometry of monomeric and dimeric aluminium
+tribromide from gas phase electron diffraction. Chem Phys Lett 257:321-326.
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Al Br 2.229(5)
+
+# Bond lengths: re(M)(a,b)     (angstroms)
+
+Al Br 2.216(8)
+
+NAME = AlCl3
+# Point Group: D3h
+# Shape: Trigonal planar
+# Bond lengths: rg     (angstroms)
+# Source Paper: Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
+electron diffraction and computation. Struct Chem 23 (3):879-893 
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Al Cl 2.073(4)
+
+# Bond lengths: re(M)  (angstroms)
+
+Al Cl 2.061(5)
+
+NAME = AlF3 
+# Point Group: D3h
+# Shape: Trigonal planar
+# Bond lengths: rg(a)     (angstroms)
+# Source Paper: a. Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
+electron diffraction and computation. Struct Chem 23 (3):879-893
+b. Hargittai M, Kolonits M, Tremmel J, Fourquet J-L, Ferey G (1990) The molecular geometry of iron trifluoride
+from electron diffraction and a reinvestigation of aluminum trifluoride. Struct Chem 1:75-78 
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Al F 1.632(3)
+
+# Bond lengths: re(M)(a,b)  (angstroms)
+
+Al F 1.621(4)
+
```

Comment:
You included AlBr3, AlCl3, and AlF3, but seem to have skipped AlI3 which was next in the book?